### ISSUE TITLE
Preview server does not detect other index files (index.xml)

### ISSUE THREAD

**Issue by clairernovotny** [statiqdev/Statiq#893]
The preview web server isn't detecting & serving an xml index file called index.xml. I'd expect that to be served implicitly.

**Comment by daveaglick** [comment]
Yeah, that should probably be served by default. Root cause is that Kestrel won't serve any content types it doesn't know about. Try adding `--content-type xml=text/xml` on the CLI and see if it works for now.

I'll add XML as a default for the next release. I've also got a long content type mapping dictionary now that I didn't have when I wrote that part - since it's a local server, maybe I'll just enable _every_ content type by default.

**Comment by clairernovotny** [comment]
The specific use case here is an RSS feed where the path is `/api/rss`
